benchmarks/project/src/main/java/io/camunda/zeebe/Worker.java

```diff
@@ -44,8 +46,12 @@ public void run() {
     final var variables = readVariables(workerCfg.getPayloadPath());
     final BlockingQueue<Future<?>> requestFutures = new ArrayBlockingQueue<>(10_000);
     final BlockingDeque<DelayedCommand> delayedCommands = new LinkedBlockingDeque<>(10_000);
-
     final ZeebeClient client = createZeebeClient();
+    final JobWorkerMetrics metrics =
+        JobWorkerMetrics.micrometer()
+            .withMeterRegistry(prometheusRegistry)
+            .withTags(Tags.of("WorkerName", workerCfg.getWorkerName(), "JobType", jobType))
```

Comment:
:x: Let's keep the convention for labels to be all lower case, so `workerName`, `jobType`, etc.